### PR TITLE
[KeyVault] `az keyvault key encrypt/decrypt`: add parameter `--data-type` for explicitly specifing the type of original data 

### PR DESCRIPTION
**Pull request by bim-msft**
This change has been confirmed with the service team.
Usage: `az keyvault key encrypt/decrypt --data-type base64/plaintext`
This PR also contains fixes for live tests:
1. Replace `time.sleep` with limited retry
2. Temporarily disable the RBAC tests

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/keyvault/_params.py [hunk 1]
```diff
@@ -229,10 +230,16 @@ class CLIJsonWebKeyOperation(str, Enum):
             c.argument('algorithm', options_list=['--algorithm', '-a'], arg_type=get_enum_type(JsonWebKeyEncryptionAlgorithm))
 
     with self.argument_context('keyvault key encrypt') as c:
-        c.argument('value', help='The value to be encrypted.', validator=validate_encryption)
+        c.argument('value', help='The value to be encrypted. Default data type is base64 encoded string.',
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/keyvault/_params.py:233]:
```suggestion
        c.argument('value', help='The value to be encrypted. Default data type is Base64 encoded string.',
```

File: src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py [hunk 2]
```diff
@@ -534,22 +537,30 @@ def test_keyvault_secret_soft_delete(self, resource_group):
...
+                self.cmd('keyvault secret list-deleted --vault-name {kv}', checks=self.check('length(@)', 1))
+                self.cmd('keyvault secret list-deleted --vault-name {kv} --maxresults 10',
+                         checks=self.check('length(@)', 1))
+                self.cmd('keyvault secret show-deleted --id {secret_recovery_id}',
+                         checks=self.check('id', '{secret_id}'))
+                self.cmd('keyvault secret show-deleted --vault-name {kv} -n {sec}',
+                         checks=self.check('id', '{secret_id}'))
+            except:  # pylint: disable=bare-except
+                time.sleep(10)
+                cnt_timeout += 10
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py:561]: I seems not a timeout. It is a counter.

Comment (bim-msft) [reply src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py:561]: Exactly, bad variable name. I will modify it.

**Comment by yonzhan** [comment]
KeyVault

**Pull request merged**